packages/devtools_app/lib/src/extensions/extension_screen.dart

```diff
@@ -155,3 +155,10 @@ class EmbeddedExtensionHeader extends StatelessWidget {
     );
   }
 }
+
+extension ExtensionConfigExtension on DevToolsExtensionConfig {
+  IconData get icon => IconData(
```

Comment:
Why does this need to be an extension instead of being a property of `DevToolsExtensionConfig`?

Reply:
Makes sense, thanks for explaining!